### ISSUE TITLE
"about" is hardcoded?

### ISSUE THREAD

**Issue by regular** [ssbc/ssb-social-index#7]
Here the string 'about' is harcoded. https://github.com/ssbc/ssb-social-index/blob/23beb7f11b02f511d8e7415b250003dc7b7138d6/index.js#L227

Can this be correct? I struggle to use this module for a different purpose (not using "about" as destField). Can this be the cause?

**Comment by staltz** [comment]
Yeah we should remove the hard coding. It should be using `Buffer.from(options.type)`
